lib/app/sequences/03_patient_standalone_launch_sequence.rb

```diff
@@ -0,0 +1,237 @@
+module Inferno
+  module Sequence
+    class PatientStandaloneLaunchSequence < SequenceBase
```

Comment:
Remove 'Patient'?

Reply:
I'll do this after. This needs more testing and isn't visible on the web interface.